tmt/steps/prepare/feature/__init__.py

```diff
@@ -126,23 +150,20 @@ def go(
         if self.opt('dry'):
             return []
 
-        # Enable or disable epel
-        for feature_key in _FEATURES:
-            value = cast(Optional[str], getattr(self.data, feature_key, None))
+        for feature_id in _FEATURE_PLUGIN_REGISTRY.iter_plugin_ids():
+            feature = find_plugin(feature_id)
```

Comment:
I'd probably use `plugin_id` and `plugin`. `feature` as a name is very common in this file and it may be unclear what exactly is it labeling, but `plugin_id` and `plugin` would be clear.